# HG changeset patch
# Parent 2bc60167b9f8791747d2cea09c5207a9c2d7ef7b
build: detect is libiconv is present

Detect if libiconv is present in the system, since we will have to
link against it when using iconv.

Signed-off-by: Roger Pau Monne <roger.pau@entel.upc.edu>

diff -r 2bc60167b9f8 Config.mk
--- a/Config.mk	Tue Dec 20 05:25:19 2011 +0100
+++ b/Config.mk	Tue Dec 20 07:52:38 2011 +0100
@@ -9,6 +9,10 @@
 XEN_OS              ?= $(shell uname -s)
 
 CONFIG_$(XEN_OS) := y
+CONFIG_LIBICONV     := $(shell export OS="`uname -s`"; \
+                       export CHECK_LIB="$(CHECK_LIB)"; \
+                       . $(XEN_ROOT)/tools/check/funcs.sh; \
+                       has_lib libiconv.so && echo 'y' || echo 'n')
 
 SHELL     ?= /bin/sh
